test/path_resolver_test.cr

```diff
@@ -53,6 +53,44 @@ module Shards
       assert_raises(Error) { resolver("unknown").install }
     end
 
+    def test_installed_reports_if_library_is_linked
```

Comment:
Can you write a few targeted tests, instead of a big one that tests many things?

Reply:
sure